src/tx/builder/helpers.js

```diff
@@ -106,6 +104,22 @@ export function commitmentHash (name, salt = createSalt()) {
   return `cm_${encodeBase58Check(hash(Buffer.concat([Buffer.from(name.toLowerCase()), formatSalt(salt)])))}`
 }
 
+// based on https://github.com/aeternity/protocol/blob/master/node/api/api_encoding.md
+const base64Types = ['ba', 'cb', 'or', 'ov', 'pi', 'ss', 'cs', 'ck', 'cv', 'st', 'tx']
+const base58Types = ['ak', 'bf', 'bs', 'bx', 'ch', 'cm', 'ct', 'kh', 'mh', 'nm', 'ok', 'oq', 'pp', 'sg', 'th']
+// TODO: add all types with a fixed length
+const typesLength = {
+  ak: 32,
+  ct: 32,
+  ok: 32
+}
```

Comment:
Could be moved to `schema.js` file in the same directory. schema file holds all the constants as of now.

Reply:
I would like to keep it local because it is not intended to be used somewhere else. Also putting it to `schema.js` will make it a part of public api, making it more difficult to change it.